contracts/test/TestCreateFromContract.sol

```diff
@@ -15,8 +15,9 @@ contract TestCreateFromContract is ERC721, Ownable {
         _tableland = ITablelandTables(registry);
     }
 
-    function create(string memory name) public payable returns (uint256) {
```

Comment:
@joewagner did you need this return here? looked like it wasn't used so i removed it

Reply:
Nope, was left over from an earlier iteration